language/move-vm/runtime/src/data_cache.rs

```diff
@@ -187,6 +187,8 @@ impl<'r, 'l, S: MoveResolver> DataStore for TransactionDataCache<'r, 'l, S> {
                 Ok(None) => GlobalValue::none(),
                 Err(err) => {
                     let msg = format!("Unexpected storage error: {:?}", err);
+                    // REVIEW: better way to get info out of a PartialVMError?
```

Comment:
Is this intended to be resolved before this is landed or more as a TODO?

Reply:
This is actually a copy/paste error--will fix.